### PR TITLE
Feat/electron refactor

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

**Review by BotellaA — APPROVED** [review]

File: composables/runFunctionIfCloudRunning.js
[the remark this answers is not in the document]
Comment (JulienChampagnol) [reply composables/runFunctionIfCloudRunning.js:1]: Indeed

File: stores/viewer.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import _ from "lodash"
 import vtkWSLinkClient from "@kitware/vtk.js/IO/Core/WSLinkClient"
 import "@kitware/vtk.js/Rendering/OpenGL/Profiles/Geometry"
+import { connectImageStream } from "@kitware/vtk.js/Rendering/Misc/RemoteView"
```
[the remark this answers is not in the document]
Comment (JulienChampagnol) [reply stores/viewer.js:4]: Wrong copy/paste

File: test/components/Launcher.nuxt.test.js [hunk 2]
```diff
@@ -15,22 +15,22 @@
 //   directives,
 // })
 
-// const cloud_store = use_cloud_store()
+// const infra_store = use_infra_store()
```
[the remark this answers is not in the document]
Comment (JulienChampagnol) [reply test/components/Launcher.nuxt.test.js:18]: Some tests that will need to be tested but this is difficult to test

**Pull request merged**